app/controllers/course_instances_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CourseInstancesController < ApplicationController
+  # GET /course_instances/autocomplete.json
+  def autocomplete
+    @course_instances = CourseInstance.search(params[:term], current_user, params[:current_date])
+    respond_to do |format|
+      format.json do
+        render json: (@course_instances.map do |ci|
+          {
+            label: "#{ci.course.long_string} (#{ci.meetings.first.try(:schedule) || 'TBA'})",
```

Comment:
Freshman Sages are the only ones I've found. They have 4th hours which are a separate meeting.